sui_core/src/authority/authority_store.rs

```diff
@@ -105,6 +107,9 @@ pub struct SuiDataStore<const ALL_OBJ_VER: bool, const USE_LOCKS: bool, S> {
     /// by a single process acting as consensus (light) client. It is used to ensure the authority processes
     /// every message output by consensus (and in the right order).
     last_consensus_index: DBMap<u64, ExecutionIndices>,
+
+    /// Map from each epoch ID to the epoch information.
+    epochs: DBMap<EpochId, EpochInfo>,
```

Comment:
Here:
- should we store a certificate for the epoch rather than just info.
- should each epoch be signed by the committee of the previous epoch?

Reply:
Interesting. I was thinking to use `EpochInfo` only for each validator locally to facilitate epoch changes. Haven't thought about the need to send it around. Do we need to send it around？